src/common/engine.js

```diff
@@ -322,6 +326,7 @@ export class CurrencyEngine {
       }
       // add transaction to list of tx's, and array of changed transactions
       this.transactionList[currencyCode].push(edgeTransaction)
+      this.walletLocalData.numTransactions[currencyCode]++
```

Comment:
Safer to just set the number of transactions equal to the length of the transaction list array. Incrementing should work but just in case the count is out of sync, using the length will put it back in sync.